instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -271,12 +291,27 @@ def process_response(self, request, response):
         if activation and span:
             if is_asgi_request:
                 set_status_code(span, response.status_code)
+                if span.is_recording() and span.kind == SpanKind.SERVER:
+                    # asgi_getter inside asgi_collect_custom_response_attributes
```

Comment:
I don't believe you have to explain this reasoning behind this, so I would remove the comments.